hybridse/include/vm/catalog.h

```diff
@@ -358,7 +358,7 @@ class PartitionHandler : public TableHandler {
     virtual std::unique_ptr<WindowIterator> GetWindowIterator() = 0;
 
     /// Return HandlerType::kPartitionHandler by default
-    const HandlerType GetHanlderType() override { return kPartitionHandler; }
+    const HandlerType GetHandlerType() override { return kPartitionHandler; }
```

Comment:
pls update method calling as well

Reply:
Can u explain in which file is calling method?

Reply:
<img width="799" alt="Screen Shot 2022-05-25 at 5 18 38 PM" src="https://user-images.githubusercontent.com/11715200/170228379-12164f93-0832-4fef-bd43-dae228851b7d.png">